Common/FeatureFlags.swift

```diff
@@ -114,10 +116,24 @@ struct FeatureFlagConfiguration: Decodable {
         #endif
 
         // Swift compiler config is inverse, since the default state is enabled.
-        #if WALSH_INSULIN_MODEL_DISABLED
-        self.walshInsulinModelEnabled = false
+        #if SIRI_DISABLED
+        self.siriEnabled = false
         #else
-        self.walshInsulinModelEnabled = true
+        self.siriEnabled = true
+        #endif
+        
+        // Swift compiler config is inverse, since the default state is enabled.
```

Comment:
Shouldn't the default state be `disabled` for this?

Reply:
Why? This is a feature I'd assume we'd want in all builds, unless it needs to be disabled for some reason (i.e. Tidepool Loop).